ReactWindows/ChakraBridge/JsModulesUnbundle.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdint.h>
+
+struct JsModulesUnbundleModule sealed
+{
+public:
+	wchar_t* source;
+	wchar_t* sourceUrl;
+	~JsModulesUnbundleModule()
+	{
+		delete[] source;
+		delete[] sourceUrl;
+	}
+};
+
+class JsModulesUnbundle
+{
+public:
+    JsModulesUnbundle(const wchar_t* sourcePath);
+	~JsModulesUnbundle();
+	virtual JsModulesUnbundleModule* GetModule(uint32_t index);
+	virtual wchar_t* GetStartupCode();
```

Comment:
In this case, yes.

---
In reply to: [103549316](https://github.com/ReactWindows/react-native-windows/pull/1017#discussion_r103549316) [](ancestors = 103549316)